traverson4j-core/src/main/java/uk/co/autotrader/traverson/http/Response.java

```diff
@@ -34,6 +35,14 @@ public void setResource(T resource) {
         this.resource = resource;
     }
 
+    public String getError() {
```

Comment:
For client code, how will they know if its an error situation, do you propose they check for non-nullness here or the successful method?
Any thoughts on this being a JSONObject instead of a string? What will clients do with this?